### ISSUE TITLE
error finding in training

### ISSUE THREAD

**Issue by Aoshika123** [iCVTEAM/PART#1]
Hello, thank you for your work. I modified the batchsize to 12 before training, and then error finding occurred after a period of time. Did the author encounter this problem before? Is it because of the lr setting problem?

**Comment by zhao1f** [comment]
Hi, I have tested it on several different machines and found it works well.  Besides the commonly occurring Nan training in other codes, one possible problem here might be the abnormal values in the object part discovery.  Slightly modifying the learning rate or batch size may help this issue. 

If these do not help, modifying the hyper-parameters of part discovery (part number and queue number)  may help.

**Comment by Carinazhao22** [comment]
Did you solve the problem? I also met the same problem.

**Comment by zhao1f** [comment]
Hi, modifying the hyper-parameters of part discovery (part number and queue number) may help.
